Source/Lib/Codec/EbPictureDecisionProcess.c

```diff
@@ -664,7 +664,7 @@ EB_ERRORTYPE SignalDerivationMultiProcessesOq(
 	  pictureControlSetPtr->cu16x16Mode = CU_16x16_MODE_0;
 
     // Set Skip OIS 8x8 Flag
-    pictureControlSetPtr->skipOis8x8 = EB_FALSE;
+    pictureControlSetPtr->skipOis8x8 = (pictureControlSetPtr->sliceType == EB_I_PICTURE && (pictureControlSetPtr->encMode <= ENC_MODE_10)) ? EB_FALSE : EB_TRUE;
```

Comment:
why adding the logic of "pictureControlSetPtr->sliceType == EB_I_PICTURE"?

Reply:
I guess just want to speed up for M11, but need to do OIS for intra all the time